assets/js/pages/main.js

```diff
@@ -0,0 +1,3 @@
+import getHeaderRender from '../components/headerTemplate.js';
+import onDOMReady from '../utils/domReady.js';
+onDOMReady(() => {getHeaderRender();});
```

Comment:
화살표 함수는 이렇게 간략히 쓸 수도 있습니다.
```suggestion
onDOMReady(()=>getHeaderRender());
```

Reply:
또는 이렇게도 가능해요.
```suggestion
onDOMReady(getHeaderRender);
```